{"uid":"74cdb548c73d1661","name":"修改收款单成功","fullName":"test_case_fundsmanagement_receipt.TestCasereceipt#test_4_change_customer","historyId":"dd9c30586c68e7d1151f6e8eecc2898f","time":{"start":1673079402907,"stop":1673079435048,"duration":32141},"description":"基础设置-新增收款单用例","descriptionHtml":"<p>基础设置-新增收款单用例</p>\n","status":"failed","statusMessage":"pytest_assume.plugin.FailedAssumption: \n1 Failed Assumptions:\n\ntest_case_fundsmanagement_receipt.py:38: AssumptionFailure\n>>\tpytest.assume(self.lp == udata['message'])\nAssertionError:","statusTrace":"tp = <class 'pytest_assume.plugin.FailedAssumption'>, value = None, tb = None\n\n    def reraise(tp, value, tb=None):\n        try:\n            if value is None:\n                value = tp()\n            if value.__traceback__ is not tb:\n>               raise value.with_traceback(tb)\nE               pytest_assume.plugin.FailedAssumption: \nE               1 Failed Assumptions:\nE               \nE               test_case_fundsmanagement_receipt.py:38: AssumptionFailure\nE               >>\tpytest.assume(self.lp == udata['message'])\nE               AssertionError:\n\nD:\\Pycharm\\Python\\venv\\lib\\site-packages\\six.py:718: FailedAssumption","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_session_faker","time":{"start":1673079240499,"stop":1673079240611,"duration":112},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false},{"name":"init_driver","time":{"start":1673079240611,"stop":1673079263655,"duration":23044},"status":"passed","steps":[],"attachments":[{"uid":"9c7fd1dfa17f0f98","name":"点击('xpath', '//*[@id=\"cancelAccountContainer\"]/div[1]/div/div[1]/div/div/button/i')选择企业后","source":"9c7fd1dfa17f0f98.png","type":"image/png","size":77631},{"uid":"faf0648fc3090edc","name":"点击('xpath', '//*[@id=\"cancelAccountContainer\"]/div[1]/div/div[1]/div/div/button/i')进入账套后","source":"faf0648fc3090edc.png","type":"image/png","size":87315},{"uid":"dc913c3534d11c93","name":"对('xpath', '//*[@id=\"cancelAccountContainer\"]/div[2]/div[1]/div[1]/ul/li[3]/button/span/span')打开进销存后","source":"dc913c3534d11c93.png","type":"image/png","size":87878}],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":3,"stepsCount":0,"hasContent":true},{"name":"Receipt","time":{"start":1673079263655,"stop":1673079272074,"duration":8419},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false}],"testStage":{"description":"基础设置-新增收款单用例","status":"failed","statusMessage":"pytest_assume.plugin.FailedAssumption: \n1 Failed Assumptions:\n\ntest_case_fundsmanagement_receipt.py:38: AssumptionFailure\n>>\tpytest.assume(self.lp == udata['message'])\nAssertionError:","statusTrace":"tp = <class 'pytest_assume.plugin.FailedAssumption'>, value = None, tb = None\n\n    def reraise(tp, value, tb=None):\n        try:\n            if value is None:\n                value = tp()\n            if value.__traceback__ is not tb:\n>               raise value.with_traceback(tb)\nE               pytest_assume.plugin.FailedAssumption: \nE               1 Failed Assumptions:\nE               \nE               test_case_fundsmanagement_receipt.py:38: AssumptionFailure\nE               >>\tpytest.assume(self.lp == udata['message'])\nE               AssertionError:\n\nD:\\Pycharm\\Python\\venv\\lib\\site-packages\\six.py:718: FailedAssumption","steps":[],"attachments":[{"uid":"40e4a3088b505de8","name":"点击末张","source":"40e4a3088b505de8.png","type":"image/png","size":65286},{"uid":"d4d44150f6ccc4ac","name":"点击金额","source":"d4d44150f6ccc4ac.png","type":"image/png","size":64977},{"uid":"c73ba58307ad7353","name":"点击金额","source":"c73ba58307ad7353.png","type":"image/png","size":65445},{"uid":"36d9d05d790fbf6c","name":"清空金额","source":"36d9d05d790fbf6c.png","type":"image/png","size":65445},{"uid":"53227b527ea6741d","name":"输入金额","source":"53227b527ea6741d.png","type":"image/png","size":65827},{"uid":"11dfe25db118084d","name":"点击自动核销","source":"11dfe25db118084d.png","type":"image/png","size":65887},{"uid":"68e9834698e5148c","name":"点击保存","source":"68e9834698e5148c.png","type":"image/png","size":69670},{"uid":"26cd8d64d3e07500","name":"失败的截图","source":"26cd8d64d3e07500.png","type":"image/png","size":66656},{"uid":"b8109fabda5a765a","name":"断言信息","source":"b8109fabda5a765a.txt","type":"text/plain","size":42},{"uid":"b91677ca29ad2ef7","name":"失败截图","source":"b91677ca29ad2ef7.png","type":"image/png","size":57516}],"parameters":[],"shouldDisplayMessage":true,"attachmentsCount":10,"stepsCount":0,"hasContent":true},"afterStages":[{"name":"init_driver::0","time":{"start":1673079470176,"stop":1673079474429,"duration":4253},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false}],"labels":[{"name":"epic","value":"资金管理"},{"name":"story","value":"修改收款单名称"},{"name":"feature","value":"收款单模块"},{"name":"tag","value":"@pytest.mark.usefixtures('Receipt')"},{"name":"suite","value":"test_case_fundsmanagement_receipt"},{"name":"subSuite","value":"TestCasereceipt"},{"name":"host","value":"DESKTOP-KEJFEBN"},{"name":"thread","value":"24804-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"test_case_fundsmanagement_receipt"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"udata","value":"{'Name': '修改收款单数据01', 'legend': '修改收款单成功', 'message': '保存成功'}"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Product defects","matchedStatuses":[],"flaky":false}],"tags":["@pytest.mark.usefixtures('Receipt')"]},"source":"74cdb548c73d1661.json","parameterValues":["{'Name': '修改收款单数据01', 'legend': '修改收款单成功', 'message': '保存成功'}"]}